cpp/benchmarks/io/cuio_common.cpp

```diff
@@ -145,6 +145,8 @@ std::vector<cudf::size_type> segments_in_chunk(int num_segments, int num_chunks,
 // Executes the command and returns stderr output
 std::string exec_cmd(std::string_view cmd)
 {
+  // Prevent the output from the command from mixing with the original process' output
+  fflush(nullptr);
```

Comment:
Is this `std::fflush`? Can we use the `std::` prefix if so?

https://en.cppreference.com/w/cpp/io/c/fflush

Reply:
May also need `#include <cstdio>`.

Reply:
Not sure if it was `std::fflush`, but now it is :)
Added the include too.